ext/nnstreamer/meson.build

```diff
@@ -6,6 +6,21 @@ if flatbuf_support_is_available
 endif
 
 if protobuf_support_is_available
+  if grpc_support_is_available
+    prog_which = find_program('which')
+    grpc_cpp_plugin_path = run_command(prog_which, 'grpc_cpp_plugin')
+    grpc_pb_gen = generator(pb_comp,
+      output: ['@BASENAME@.grpc.pb.h', '@BASENAME@.grpc.pb.cc'],
+      arguments : [
+        '--proto_path=@CURRENT_SOURCE_DIR@/include',
+        '--plugin=protoc-gen-grpc=' + grpc_cpp_plugin_path.stdout().strip(),
+        '--grpc_out=@BUILD_DIR@',
+        '@INPUT@'
+      ]
+    )
+    grpc_pb_gen_src = grpc_pb_gen.process('./include/nnstreamer.proto')
+  endif
```

Comment:
Please properly handle all 4 cases: (protobuf_support, grpc_support) = (0,0), (1,1), (0,1), (1,0), along with proper error/warning messages

Reply:
Sure, got it.